app/src/main/java/net/squanchy/notification/NotificationModule.java

```diff
@@ -16,4 +19,14 @@ NotificationService favoritesService(
     ) {
         return new NotificationService(authService, eventRepository);
     }
+
+    @Provides
+    NotificationCreator notificationCreator(Context context) {
+        return new NotificationCreator(context);
+    }
+
+    @Provides
+    Notifier notifier(Context context) {
+        return Notifier.from(context);
```

Comment:
nitpick (not a blocker but maybe next PR?) `Notifier` should not have a `from` method and you should have the dependencies explicitly in this module :tongue: